addons/website_slides/static/src/js/slides_course_join.js

```diff
@@ -49,6 +50,13 @@ var CourseJoinWidget = publicWidget.Widget.extend({
      */
     _onClickJoin: function (event) {
         var channelId = this.channelId || $(event.currentTarget).data('channel-id');
+        this.joinCourse(channelId); 
+    },
+
+    /**
+     * @public
```

Comment:
Missing `@param` for channelId.

This stuff is important because the JS doc is run through a program that builds a HTML file with it.
If there are errors, the program will fail (and the JS team will be very very unhappy).